src/storage/txn/commands/mod.rs

```diff
@@ -600,10 +623,16 @@ impl Debug for Command {
     }
 }
 
+/// Commands that do not need to modify the database during execution will implement this trait.
+/// Currently, this refers to the following variants of `storage::txn::commands::Command`:
```

Comment:
Here and below, I don't think listing the commands is a good idea - it is difficult to keep such lists up to date.

Reply:
Done.